fuse/eval/metrics/libs/survival.py

```diff
@@ -22,6 +22,8 @@ def c_index(
         """
         if isinstance(pred[0], np.ndarray):
             pred = np.asarray(pred)[:, event_class_index]
+        elif isinstance(pred, list):
+            pred = np.asarray(pred)
```

Comment:
pred is a list of what? floats? 
Should we add an assert that event_class_index is -1?

Reply:
pred should contain float score per sample.
The ground truth has two components: event/censor indicator and time of event/censor

Reply:
@mosheraboh see new fix - following your comments